docs/source/guides/worklock_guide.rst

```diff
@@ -9,6 +9,12 @@ Overview
 
 :ref:`worklock-architecture` is the distribution mechanism for the NuCypher token.
 
+.. topic:: Important
+
+    The Ethereum address for the WorkLock contract is
+    `0xe9778E69a961e64d3cdBB34CF6778281d34667c2 <https://etherscan.io/address/0xe9778e69a961e64d3cdbb34cf6778281d34667c2>`.
+    Please make sure that you interact with this contract address and no other.
+
```

Comment:
@cygnusv this part of the docs isn't displaying correctly.

Try the following (note the `important` tag and the addition of the underscore for the link):
```
.. important::

    The Ethereum address for the WorkLock contract is
    `0xe9778E69a961e64d3cdBB34CF6778281d34667c2 <https://etherscan.io/address/0xe9778e69a961e64d3cdbb34cf6778281d34667c2>`_.
    Please make sure that you interact with this contract address and no other.
```

Reply:
Whoops! I suppose we need to look closer at build artifacts.